applications/rag/frontend/main.tf

```diff
@@ -30,6 +34,46 @@ locals {
     instance_connection_name = format("%s:%s:%s", var.project_id, var.region, "pgvector-instance")
 }
 
+# IAP Section: Enabled the IAP service
+resource "google_project_service" "project_service" {
```

Comment:
can this be done in the common iap.tf module?

Reply:
We prefer here since the iap parts focus on k8s resource